config/interpolate_walk.go

```diff
@@ -54,6 +54,9 @@ type interpolationWalkerContextFunc func(reflectwalk.Location, ast.Node)
 
 func (w *interpolationWalker) Enter(loc reflectwalk.Location) error {
 	w.loc = loc
+	if loc == reflectwalk.WalkLoc {
```

Comment:
I had to look this up in the code. `WalkLoc` seems to be "the top of a given walk"